package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
```

Comment:
I copied the style from babels [example-node-server](https://github.com/babel/example-node-server/blob/master/package.json).